server/controllers/droppedAssets/handleGetQuestDetails.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from "express";
+import { errorHandler, getCredentials, getWorldDetails } from "../../utils/index.js";
+
+export const handleGetQuestDetails = async (req: Request, res: Response) => {
+  try {
+    const credentials = getCredentials(req.query);
+
+    const { dataObject } = await getWorldDetails(credentials);
+
+    return res.json(dataObject);
```

Comment:
I think we should insert the success: true in all routes or none, to keep consistency

Reply:
I'm fine with that in general but not going to add that in now. I just rebuilt the entire app and adding new success checks on all client side calls is just not something that is worth the time in this moment. Thank you for the suggestion though!